### DISCUSSION TITLE
have problems with installation umani with gatsby

### DISCUSSION THREAD

**Discussion by WatheqAlshowaiter** [umami-software/umami#832] [Q&A]
After install the pugin `gatsby-plugin-umami`
and added the config:
```
{
    resolve: `gatsby-plugin-umami`,
    options: {
      websiteId: 'UMAMI_WEBSITE_ID',
      srcUrl: 'https://umami.example.com/umami.js',
      includeInDevelopment: false,
      autoTrack: true,
      respectDoNotTrack: true
    },
  },
```
- what should I add to `websiteId`? 
- where is the login page of Umani?
- where is the page I can find the analytics?

**Comment by heriswn** [comment]
First of all, you need setups umami.

You can install umami in herokuapp (you can check in youtube tutorial). After that, you can login and access umami.

**Comment by WatheqAlshowaiter** [comment]
I have found the answer. thanks.
